translationtable/GLOBAL_TERMS.yaml

```diff
@@ -179,7 +179,6 @@
 "organization": "foaf:organization",
 "page": "foaf:page",
 "person": "foaf:Person",
-# ??? PATO_0000470: GENO:0000876
```

Comment:
Leave in and expand, @TomConlin do you recall what this comment means

Reply:
the geno  seems to be "obsolete_genetic dosage" 
the pato is  "increased amount"
I imagine the geno had been hardcoded in the source somewhere and was moved 
the tt  where it waits to be resolved  the pato may have been a SWAG at a replacement but remained insufficient to act on (requires bio input?).

There may be too much of a disconnect between terms and labels in rapidly evolving ontology to be sure the current obsolete label is relevant to the the previously hardcoded term.